server/repository/src/migrations/v1_01_11/mod.rs

```diff
@@ -0,0 +1,110 @@
+use crate::{migrations::*, StorageConnection};
+pub(crate) struct V1_01_11;
+
+impl Migration for V1_01_11 {
+    fn version(&self) -> Version {
+        Version::from_str("1.1.11")
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        // Store Tag
+        sql!(
+            connection,
+            r#"
+            CREATE TABLE store_tag (
+                id TEXT NOT NULL PRIMARY KEY,
+                store_id TEXT NOT NULL REFERENCES store(id),
+                tag_name TEXT NOT NULL
```

Comment:
i think `name` might have been enough here, not super pick on this but it's the pattern we generally use